### DISCUSSION TITLE
I've lost access to my Vercel account, amongst other issues.

### DISCUSSION THREAD

**Discussion by JGore99** [vercel/community#4684] [Account Issue]
### Question

Recently I've been trying to update a site I haven't worked on in 2 years. The domain is through Google, and it is deployed with Vercel. The last time I looked at it, everything worked great. When I push changes in GitHub, I get the error: "Failed to deploy to Production". To make matters worse, the site email, which is also tied to my Vercel login is no longer receiving emails, effectively locking me out of my Vercel dashboard and the logs. I'm sure a lot has changed with Vercel in those 2 years since my site was deployed so it is possible my site is just out of date in some capacity. I'm unsure what the issue(s) is, or where to start looking. Any suggestions on where to start, or at the very least, how to get back into my Vercel account? Thanks in advance.

**Comment by amyegan** [comment]
Sorry you're having trouble accessing your Vercel account. Since we can't safely discuss private account info here, please use the appropriate [direct channel](https://vercel.com/guides/how-to-get-vercel-support#direct-channels) found in this guide: [What is the best way to get support from Vercel?](https://vercel.com/guides/how-to-get-vercel-support#direct-channels)

**Reply by amyegan** [reply]
I'm not sure if you already tried this, but if you already connected your GitHub account then you should be able to "Continue with GitHub" from the [login](https://vercel.com/login) page instead of using your email. 

If you let me know the domain, I can look into that problem for you.

**Reply by JGore99** [reply]
Hi Amy,

The domain is: https://www.jeffgore.net/

I tried using GitHub to log in, but it looks like my accounts aren't linked.

I did reach out to registration@Vercel. Hoping to hear back soon.

-Thanks!

**Reply by JGore99** [reply]
@amyegan Just wanted to follow up on this issue. Were you able to look into this?

**Comment by cowmonkey731** [comment]
Hilariously painful customer support and user lockout handling by Vercel. Use another provider if you value not getting locked out of your account with 0 recourse to then be forced into seeking out obscure "support guides".

**Reply by amyegan** [reply]
Hey @cowmonkey731. We're not able to handle account access issues here in public since we would need to discuss private information about your account.

The email address to open a login issue ticket is currently [registration@vercel.com](mailto:registration@vercel.com). That could change in the future so I shared the [support guide](https://vercel.com/guides/how-to-get-vercel-support#direct-channels) link above to be sure anyone who comes across this discussion in the future will get the right address. If you contacted them and have not yet gotten the help you need, please let me know the case number so I can check on it for you 🙏
